src/dotnet/APIView/APIViewWeb/Repositories/DevopsArtifactRepository.cs

```diff
@@ -83,13 +91,38 @@ private string GetArtifactRestAPIForRepo(string repoName)
 
         public async Task RunPipeline(string pipelineName, string reviewDetails, string originalStorageUrl)
```

Comment:
Looks like you are using pipeline name here and are using a naming convention. Is that documented anywhere? If the naming convention is helpful we can continue with that but I will say that using names end up being less stable then definition ids.

Reply:
We fetch definition ID using pipeline name. Pipeline name follows  `tools - generate-<language>-apireview` foramt and this is generated by caller within APIView when it triggers Pipeline. I can document this as part of document I create about sandboxing and include in APIView contributing guide as well.